preprocess/pca.py

```diff
@@ -16,13 +21,14 @@ def pca(input_prefix: str, pca_config: dict, output_tag: str) -> None:
                   args_dict=pca_config)
 
     @staticmethod
-    def pc_scatterplot(input_prefix: str, output_tag: str) -> None:
+    def pc_scatterplot(tag: str) -> None:
         """ Visualises eigenvector with scatterplot [matrix] """
-        pass
-        # TODO
-        #"plotly_graph".write_html("smth_based_on_output_tag.html")
+        eigenvec = pd.read_table(f'{data_root}/pca/{tag}.eigenvec')[['IID', 'PC1', 'PC2']]
+        eigenvec.index = eigenvec.IID
+        eb_df = SplitBase({}).get_ethnic_background()
```

Comment:
you can make get_ethnic_background a static method since it doesn't require self. if you do this, you can simply call  SplitBase.get_ethnic_background() without initialising the class